### PR TITLE
Fix problem with release of egg

### PR DESCRIPTION
**Pull request by loechel**
There seems to be a problem with the last release: 

https://github.com/plone/diazo/pull/70#issuecomment-325274263

@pbauer pleased me to look at it. Maybe someone from @plone/release-team should also.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -126,7 +126,9 @@ basepython = python2.7
 deps =
     -cconstraints.txt
     zest.releaser[recommended]
+    check-manifest
```
Comment (gforcada) [inline tox.ini]: zest.releaser already uses check-manifest and as long as the @plone/release-team does the releases we are already using our own tooling (in buildout.coredev) to make the releases. Is there any real benefit to keep this here?

**Comment by pbauer** [comment]
@esteele has made a new release with that included https://github.com/plone/diazo/commit/a1fbf3d482172474c347840f7276cd1586aaab9a

File: tox.ini (hunk 1 above)
Comment (loechel) [reply tox.ini]: Ok, I have removed it

**Comment by loechel** [comment]
then we should close this pull request.

**Pull request closed without merging**